### PR TITLE
chore: finish screen feedback

### PR DESCRIPTION
**Pull request by rolznz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,32 +86,26 @@ _If you get a blank screen the first load, close the window and start the app ag
...
 
-_For cert and macaroon, either hex or file options can be used._
+_To configure via env, the following parameters must be provided:_
 
+- `LN_BACKEND_TYPE`: LND
 - `LND_ADDRESS`: the LND gRPC address, eg. `localhost:10009` (used with the LND backend)
 - `LND_CERT_FILE`: the location where LND's `tls.cert` file can be found (used with the LND backend)
 - `LND_MACAROON_FILE`: the location where LND's `admin.macaroon` file can be found (used with the LND backend)
 
-### BREEZ Backend parameters
```
Comment (im-adithya) [inline README.md:109]: Should we mention that BREEZ can only be run via the onboarding setup?

File: frontend/src/screens/setup/SetupFinish.tsx [hunk 2]
```diff
@@ -1,79 +1,81 @@
...
 }
+
+const finishSetup = async (
+  csrf: string,
+  nodeInfo: NodeInfo,
+  unlockPassword: string
+): Promise<boolean> => {
+  try {
+    if (!csrf) {
+      throw new Error("info not loaded");
```
Comment (im-adithya) [inline frontend/src/screens/setup/SetupFinish.tsx:53]: `csrf not loaded`

File: frontend/src/screens/setup/SetupFinish.tsx [hunk 3]
```diff
@@ -1,79 +1,81 @@
...
 import Container from "src/components/Container";
 import { useInfo } from "src/hooks/useInfo";
 import useSetupStore from "src/state/SetupStore";
 import { useCSRF } from "src/hooks/useCSRF";
 import { handleRequestError } from "src/utils/handleRequestError";
 import { request } from "src/utils/request";
 import Loading from "src/components/Loading";
+import { NodeInfo } from "src/types";
 
+let hasFetched = false;
```
Comment (im-adithya) [inline frontend/src/screens/setup/SetupFinish.tsx:12]: Why can't this be a state variable?

File: frontend/src/state/SetupStore.ts [hunk 4]
```diff
@@ -2,16 +2,16 @@ import { NodeInfo } from "src/types";
...
   setUnlockPassword(unlockPassword: string): void;
 }
 
 const useSetupStore = create<SetupStore>((set) => ({
-  nodeInfo: null,
+  nodeInfo: {},
   unlockPassword: "",
-  setNodeInfo: (nodeInfoPartial) =>
+  updateNodeInfo: (nodeInfoPartial) =>
     set((state) => ({
```
Comment (im-adithya) [inline frontend/src/state/SetupStore.ts:15]: We still do `{ ...nodeInfoPartial } as NodeInfo` as mentioned here https://github.com/getAlby/nostr-wallet-connect-next/pull/38#discussion_r1489130405

Comment (im-adithya) [reply frontend/src/state/SetupStore.ts:15]: Got it, `NodeInfo` itself is a Partial, my bad 👍

File: frontend/src/screens/setup/SetupFinish.tsx (hunk 3 above)
Comment (rolznz) [reply frontend/src/screens/setup/SetupFinish.tsx:12]: React still calls twice if I do that. I think it's a React strict mode issue

File: frontend/src/screens/setup/SetupFinish.tsx (hunk 2 above)
Comment (rolznz) [reply frontend/src/screens/setup/SetupFinish.tsx:53]: removing instead, there was another check. Thanks

File: frontend/src/screens/setup/SetupFinish.tsx (hunk 3 above)
Comment (im-adithya) [reply frontend/src/screens/setup/SetupFinish.tsx:12]: Ah okay

File: frontend/src/state/SetupStore.ts (hunk 4 above)
Comment (rolznz) [reply frontend/src/state/SetupStore.ts:15]: oops, thanks! Changed to `nodeInfo: { ...state.nodeInfo, ...nodeInfo },`

**Comment by im-adithya** [comment]
The Finish screen keeps showing `Connecting...` even after the connection fails

File: frontend/src/screens/setup/SetupFinish.tsx (hunk 3 above)
Comment (rolznz) [reply frontend/src/screens/setup/SetupFinish.tsx]: Actually, fixed it to use a ref. Thanks!

**Pull request merged**